pitch.md

```diff
@@ -46,6 +194,28 @@ aren't and conjecture why not. If so, describe them and provide links. How well
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+I've already mentioned some of the other tools/DSLs hanging out in this domain.
+A brief list:
+   - [tkz-euclide](https://www.ctan.org/pkg/tkz-euclide?lang=en) is a LaTeX
+     package which allows for geometrically precise drawings. It is somewhat
+     cryptic and documented only in French :heart: .
+   - [sketchometry](http://start.sketchometry.org/) is a tool that allows you
+     to skethily draws things and fits your drawing to geometrically
+     significant shapes.
+   - [geogebra](https://www.geogebra.org/) is another, more conventional
+     graphical drawing aide.
+
+As discussed, all of these fail to be both precise and easy to use. I like a
+lot of what the LaTeX package does, but I think it is rather clunky to use, and
+is not well adapted to building further tooling on top of.
+
+I intend to spent some time exploring how it handles errors (I.E. intersection
+of parallel lines) to get some inspiration for how I should.
```

Comment:
I think spatial input systems actually handle this more poorly. With a spatial input system it becomes ambiguous whether the user is trying to draw parallel lines or whether they are drawing lines with similar slopes.